shared/teams/open-team/container.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import React from 'react'
+import * as I from 'immutable'
+import {MakeOpenTeamConfirm, MakeTeamClosed, type OpenTeamConfirmProps} from '.'
+import {connect} from 'react-redux'
+import {compose, withState, withPropsOnChange} from 'recompose'
+import * as Creators from '../../actions/teams/creators'
+
+type OpenTeamSettingProps = {
+  onClose: () => void,
+  routeProps: I.RecordOf<{actualTeamName: string}>,
```

Comment:
This is actually a Map, but if I typed it as such, we'd lose the information of what should be in the routeProps